docs/application/dotnet/guides/nui/nui-components/Scrollbar.md

```diff
@@ -10,28 +10,32 @@ When entering the screen consists of list, the vertical or horizontal scroll bar
 ![CreateWithProperties](./media/scrollbar_properties.PNG)
 
 ## Create with Properties

```

Comment:
Query:

This sentence should be "Create with Property" as we are using 'Property' in the other sentences too. Any particular reason to use 'Properties' here? Please let us know to clarify the same.